docs/components/marketing/EndCta.tsx

```diff
@@ -33,7 +33,6 @@ export function EndCta({ grad = 'grad1', ...props }: EndCtaProps) {
           display: 'flex',
           gap: '1rem',
           justifyContent: 'center',
-          gap: '1rem',
```

Comment:
I think you accidentally removed the gap here. Need it or it looks like this: 
<img width="654" alt="image" src="https://user-images.githubusercontent.com/1266923/123100830-cddd6480-d476-11eb-94c1-ee244673cb0a.png">

Reply:
That's really weird, I removed this because it's repeated (see line 34)

Reply:
Oh didn't see the duplication. Was a long day :)